python_submitty_utils/submitty_utils/dateutils.py

```diff
@@ -56,14 +56,23 @@ def read_submitty_date(s):
         raise SystemExit("ERROR: unexpected date format %s" % s)
     thedatetime = str(words[0] + ' ' + words[1])
     try:
+        # hoping to find timezone -0400
         with_timezone = datetime.strptime(thedatetime, '%Y-%m-%d %H:%M:%S%z')
     except ValueError:
         try:
+            # hoping to find no timezone
             without_timezone = datetime.strptime(thedatetime, '%Y-%m-%d %H:%M:%S')
             my_timezone = get_timezone()
             with_timezone = my_timezone.localize(without_timezone)
         except ValueError:
-            raise SystemExit("ERROR:  invalid date format %s" % s)
+            try:
+                # hoping to find timezone -04
```

Comment:
Where are we getting a timezone as -04? Is this for something coming from Git?

Reply:
It appeared when I was doing git stuff, I couldn't stop to figure out where it was coming from.  I think it was non repo gradeables as well.